apps/frontend/app/(website)/studio/features/modGPT/PromptPanel/ClearHistoryButton.tsx

```diff
@@ -0,0 +1,19 @@
+import { ArrowElbowDownLeft } from "@phosphor-icons/react";
+import { Trash as TrashIcon } from "@phosphor-icons/react/dist/csr/Trash";
```

Comment:
if you have some spare time, let's make sure we use icons from one single package and remove the other one.
shadcn that we are using in studio uses lucide icons by default, some other places do use it too. phosphor icons were brought to us by Tinloof i believe, if we like them more, let's replace all the other usages.